src/TokensList.php

```diff
@@ -107,6 +107,26 @@ public function getNext()
 
         return null;
     }
+    
+    /**
+     * Gets the previous token. Skips any irrelevant token (whitespaces and
+     * comments).
+     *
+     * @return Token|null
+     */
+    public function getPrevious()
```

Comment:
```suggestion
    public function getPrevious(): ?Token
```
It should work, not sure for 7.1 but let's see
That would make the phpdoc line useless